scripts/language_model/model/qa.py

```diff
@@ -0,0 +1,293 @@
+"""XLNetForQA models."""
```

Comment:
Should this be added after or as part of https://github.com/dmlc/gluon-nlp/pull/1012?